cmd/cluster/log-exporter/query_log_exporter.go

```diff
@@ -0,0 +1,402 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package query_log_exporter
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	"github.com/yugabyte/ybm-cli/internal/formatter"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+var ClusterName string
+
+var DbQueryLoggingCmd = &cobra.Command{
+	Use:   "db-query-logging",
```

Comment:
Please keep the command consistent with DB Audit logging.
DB Audit log has command : `db-audit-logs-exporter`.

CC: @Siddarth-Baldwa @bansal01yash

Reply:
@atharva-yb talked to the PMs about this. We need to keep `db-audit-export` commands under the cluster command as is being done for pg logs (in this PR)

Reply:
True. I was referring the exact word of command.
If we are going to use `ybm cluster db-audit-logs-exporter` for DB audit, we should use `ybm cluster db-query-logs-exporter` for Query logs.
If we want to use `ybm cluster db-query-logging` for Query then we need to modify `ybm cluster db-audit-logging` for Audit.
Let's keep it consistent.

Reply:
@bhupendray-yb Can you make it `db-query-log-exporter`
@atharva-yb since we need to slightly refactor audit to be under cluster anyways, then we can rename audit to `db-audit-log-exporter` . Just to be a little consistent with API. 
Note: change from logs -> log

Reply:
Any command works. Just make them similar for Audit and Query. 👍